docs/sphinx/developers/cpp/overview.txt

```diff
@@ -129,12 +129,19 @@ BSD Ports
   ``pkg install devel/boost-all devel/cmake science/hdf5 graphics/png lang/python textproc/py-genshi graphics/tiff textproc/xerces-c3 devel/git devel/googletest math/glm devel/qt5 graphics/graphviz devel/apache-ant java/openjdk7 textproc/py-sphinx print/texlive-full``
 Debian/Ubuntu
   ``apt-get install build-essential libboost-all-dev cmake libhdf5-dev libpng12-dev python python-genshi libtiff5-dev libxerces-c-dev git libgtest-dev libglm-dev qt5-default libqt5-opengl5-dev libqt5-svg5-dev graphviz ant ant-contrib ant-optional openjdk-7-jdk openjdk-7-jre python-sphinx texlive-full``
+
+Partial quick starts
+^^^^^^^^^^^^^^^^^^^^
+
+Homebrew and RedHat/CentOS do not provide packages for everything you need. The lists below will install
+*most* of the dependencies, but you will need to see the details below to install further dependencies.
```

Comment:
I was copying the use of "your" from https://github.com/openmicroscopy/bioformats/pull/1879/files#diff-001920d55adf9e9dc69e9c467845094dR126 -- happy to use whatever style, really.